addons/web_editor/static/src/js/backend/html_field.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="web_editor.HtmlField" owl="1">
+        <t t-if="props.readonly || props.notEditable">
+            <t t-if="this.showIframe">
+                <iframe t-ref="iframe" t-att-class="{'d-none': !this.state.iframeVisible, 'o_readonly': true}"></iframe>
+            </t>
+            <t t-else="">
+                <div  t-ref="readonlyElement" class="o_readonly" t-out="markupValue" />
+            </t>
+        </t>
+        <t t-else="">
+            <t t-if="state.showCodeView">
+                <textarea t-ref="codeView" class="o_codeview" t-att-value="markupValue"/>
+                <div t-ref="codeViewButton" id="codeview-btn-group" class="btn-group" t-on-click="toggleCodeView">
+                    <button class="o_codeview_btn btn btn-primary">
+                        <i class="fa fa-code" />
+                    </button>
+                </div>
+            </t>
+            <t t-else="">
+                <HtmlFieldWysiwygAdapterComponent Component="this.Wysiwyg"
+                    startWysiwyg.bind="this.startWysiwyg"
```

Comment:
What ? Is this even a valid attribute ? GitHub doesn't color it in purple like other attributes.